demo/admin/src/App.tsx

```diff
@@ -57,79 +64,81 @@ class App extends React.Component {
 
     public render(): JSX.Element {
         return (
-            <ApolloProvider client={apolloClient}>
-                <CurrentUserProvider>
-                    <SitesConfigProvider
-                        value={{
-                            configs: config.sitesConfig,
-                            resolveSiteConfigForScope: (configs, scope: ContentScope) => configs[scope.domain],
-                        }}
-                    >
-                        <DamConfigProvider
+            <ContentGenerationServiceProvider config={{ image: true, imageAdvanced: true }}>
```

Comment:
I'd probably not add this as the first provider but rather somewhere deeper in the tree. Since in a real life example you might need the `scope` for the provider config.